packages/fogbender-element/index.html

```diff
@@ -55,6 +55,7 @@
                 <fogbender-is-configured>
                   <template data-is-configured>
                     <fogbender-headless-widget />
+                    <fogbender-unread-badge />
```

Comment:
let's add new scenario and there just use "fogbender-headless-widget" and "fobender-unread-badge" (no fogbender-floating-widget)

Reply:
Acknowledged

Reply:
Done